recipe/meta.yaml

```
@@ -45,39 +55,42 @@ outputs:
         - make  # [unix]
       host:
         - zlib
+        - gmp
       run:
         - zlib
     test:
       commands:
-        - test -e $PREFIX/lib/libcryptominisat5.5.6.dylib    # [osx]
-        - test -e $PREFIX/lib/libcryptominisat5.so.5.6       # [linux]
+        - test -e $PREFIX/lib/libcryptominisat5.5.12.dylib    # [osx]
+        - test -e $PREFIX/lib/libcryptominisat5.so.5.12       # [linux]
         - IF NOT EXIST %LIBRARY_BIN%\cryptominisat5win.dll exit 1       # [win]
-        - cryptominisat5_simple --zero-exit-status <unsat.cnf
-        - cryptominisat5_simple --zero-exit-status <sat.cnf
-        - cryptominisat5_simple <unsat.cnf | grep UNSATISFIABLE  # [unix]
-        - cryptominisat5_simple <sat.cnf   | grep SATISFIABLE    # [unix]
-        - cryptominisat5_simple <unsat.cnf | findstr UNSATISFIABLE  # [win]
-        - cryptominisat5_simple <sat.cnf   | findstr SATISFIABLE    # [win]
+        - cryptominisat5 --zero-exit-status <unsat.cnf
+        - cryptominisat5 --zero-exit-status <sat.cnf
+        - cryptominisat5 <unsat.cnf | grep UNSATISFIABLE  # [unix]
+        - cryptominisat5 <sat.cnf   | grep SATISFIABLE    # [unix]
+        - cryptominisat5 <unsat.cnf | findstr UNSATISFIABLE  # [win]
+        - cryptominisat5 <sat.cnf   | findstr SATISFIABLE    # [win]
       files:
         - sat.cnf
         - unsat.cnf
+
   - name: pycryptosat
     build:
       number: {{ build }}
-      skip: True  # [win and vc<14]
-    script: install-py.sh   # [unix]
-    script: install-py.bat  # [win]
+      skip: True  # [py<35]
+      script:
+        - cd cryptominisat && python -m pip install . -vv --no-deps --no-build-isolation
     requirements:
       build:
         - {{ stdlib('c') }}
         - {{ compiler('cxx') }}
-        - cmake >=3.12
-        - make  # [unix]
       host:
         - python
-        - setuptools
+        - setuptools >=42
+        - pathlib
+        - tomli
         - wheel
         - pip
+        - gmp
```

```suggestion
        - gmp {{ gmp }}
```
For maintainability, see https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L321-L322